unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BlockerController/Tests/BlockerHandlerCan.cs

```diff
@@ -0,0 +1,119 @@
+using DCL.Controllers;
+using DCL.Helpers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+using DCL;
+using System.Linq;
+using NSubstitute;
+using NUnit.Framework;
+
+public class BlockerHandlerCan
+{
+    private IBlockerInstanceHandler blockerInstanceHandler;
+    private IBlockerAnimationHandler blockerAnimationHandler;
+    private ISceneHandler sceneHandler;
+    private GameObject blockersParent;
+
+    [SetUp]
+    protected void SetUp()
+    {
+        RenderProfileManifest.i.Initialize();
+
+        sceneHandler = Substitute.For<ISceneHandler>();
+
+        var allLoadedParcelCoords = new HashSet<Vector2Int>();
+        allLoadedParcelCoords.Add(new Vector2Int(0, 0));
+        allLoadedParcelCoords.Add(new Vector2Int(-1, 0));
+        allLoadedParcelCoords.Add(new Vector2Int(-1, 1));
+
+        sceneHandler.GetAllLoadedScenesCoords().Returns(allLoadedParcelCoords);
+
+        blockerAnimationHandler = Substitute.For<IBlockerAnimationHandler>();
+
+        //NOTE(Brian): Call OnFinish() when blockerAnimationHandler.FadeOut is called. 
+        blockerAnimationHandler.FadeOut(Arg.Any<GameObject>(), Arg.Invoke());
+
+        blockerInstanceHandler = new BlockerInstanceHandler(new DCLCharacterPosition(), blockerAnimationHandler);
+
+        blockersParent = new GameObject();
```

Comment:
You can use `TestBase.CreateTestGameObject(name)` and it will create an object that automatically will be destroyed so you dont have to teardown them. It's a matter of opinion though, if you prefer it explicitly in the test itself, that's fine.

Reply:
Interesting, actually I didn't know we had that method.

Reply:
I can't use this because this class doesn't inherit from `TestsBase`. Going to resolve this and maybe we can put this later un `TestHelpers` somehow.